lib/Models/Subscriber.php

```diff
@@ -38,6 +38,9 @@ class Subscriber extends Model {
   const STATUS_BOUNCED = 'bounced';
   const STATUS_INACTIVE = 'inactive';
 
+  const DEPRECATED_LINK_TOKEN_LENGTH = 6;
```

Comment:
Shall we add a comment describing why is this deprecated? Why is it deprecated anyway? I thought that deprecated is something that can be removed eventually. Can this be removed?

Reply:
What name do you suggest? What about OLD_LINK_TOKEN_LENGTH ?

Reply:
Maybe `OBSOLETE_TOKENS_LENGTH`?